test/fv/crud.js

```diff
@@ -11,7 +11,7 @@ const expect = chai.expect;
 const utils = require('./utils');
 const {SHORT_STEP, MED_STEP, LONG_STEP} = utils.TIMEOUTS;
 
-describe('Chaincode CRUD', () => {
+describe.only('Chaincode CRUD', () => {
```

Comment:
I guess this isn't deliberate?

Reply:
it was... to make the build faster to run.. but need to remove.